### PR TITLE
SC-1888 : Added Content Import Api

### PR DESCRIPTION
**Pull request by krgauraw**

### CODE REVIEW

File: content-api/content-actors/src/main/scala/org/sunbird/content/import/ImportManager.scala [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    val processId: String = UUID.randomUUID().toString
+    val invalidCodes: util.List[String] = new util.ArrayList[String]()
+    validateAndGetRequest(reqList, processId, invalidCodes).map(list => {
+      if (CollectionUtils.isNotEmpty(invalidCodes)) {
+        val msg = if (invalidCodes.asScala.filter(c => StringUtils.isNotBlank(c)).toList.size > 0) " | Required Property's Missing For " + invalidCodes else ""
+        throw new ClientException(ImportErrors.ERR_REQUIRED_PROPS_VALIDATION, ImportErrors.ERR_REQUIRED_PROPS_VALIDATION_MSG + REQUIRED_PROPS + msg)
+      } else {
+        list.asScala.map(req => {
+          val source: String = req.getOrDefault(ImportConstants.SOURCE, "").toString
+          val identifier = if(StringUtils.isNotBlank(source)) source.substring(source.lastIndexOf('/') + 1) else Identifier.getIdentifier(graphId, Identifier.getUniqueIdFromTimestamp)
```
Comment (maheshkumargangula) [inline content-api/content-actors/src/main/scala/org/sunbird/content/import/ImportManager.scala:39]: Why are we using this identifier? It should create a new always.

File: content-api/content-actors/src/main/scala/org/sunbird/content/import/ImportManager.scala [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        val response = ResponseHandler.OK()
+        response.put(ImportConstants.PROCESS_ID, processId)
+        response
+      }
+    })
+
+  }
+
+  def validateAndGetRequest(reqList: util.List[util.Map[String, AnyRef]], processId: String, invalidCodes: util.List[String])(implicit oec: OntologyEngineContext, ec: ExecutionContext): Future[util.List[util.Map[String, AnyRef]]] = {
+    Future {
```
Comment (maheshkumargangula) [inline content-api/content-actors/src/main/scala/org/sunbird/content/import/ImportManager.scala:54]: Why Future?

Comment (krgauraw) [reply content-api/content-actors/src/main/scala/org/sunbird/content/import/ImportManager.scala]: As discussed, once we will do testing for 200-300 objects in single request. we will take a call and remove if not required.

File: content-api/content-actors/src/main/scala/org/sunbird/content/import/ImportManager.scala (hunk 1 above)
Comment (krgauraw) [reply content-api/content-actors/src/main/scala/org/sunbird/content/import/ImportManager.scala]: As discussed, it is required for bulk approval from dock to diksha.

**Comment by krgauraw** [comment]
@maheshkumargangula I have raised another pr https://github.com/project-sunbird/knowledge-platform/pull/362 with all changes suggested by you. 
closing this pr because of multiple commits.

**Pull request closed without merging**